stable/yugaware/templates/rbac.yaml

```diff
@@ -1,7 +1,8 @@
+{{ if not .Values.yugaware.serviceAccount }}
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: {{ .Release.Name }}
+  name: {{ .Values.yugaware.serviceAccount | default .Release.Name }}
```

Comment:
This won't be needed here I think. As this ServiceAccount is created only when the yugaware.serviceAccount is not set.

Reply:
Make sense.